QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150714_002062	$Despite the royal forces occupying the town, there has yet to be any threatening movement. {nl}What's going to happen to us…
+QUEST_20150714_002063	$As promised, I will cast magic on you that will protect you from Petrified Frosts. {nl}But if you move out from this region, this magic won't be as effective so be careful.
+QUEST_20150714_002064	$There has been no progress since Ruklys' group retreated to the Great Earth Fortress. {nl}The priests under Maven's direction are reluctant to fight, and the King is hiring mages by paying them a hefty sum.
+QUEST_20150714_002065	$But we expect little to no progress at all even after their participation.
+QUEST_20150714_002066	$How could this happen?! {nl}Lydia Schaffen is aiding Kadumel's forces!
+QUEST_20150714_002067	$Lydia Schaffen, a person I've only heard of in rumors, came to the city with her followers. {nl}The officers and soldiers expect she'll change the tide of the battle. {nl}His Majesty ordered us to support her to the best of our ability.
+QUEST_20150714_002068	$General Ruklys has fallen in battle, and the Great Earth Fortress has been taken. {nl}Some people say the general died to Lydia Schaffen's arrow, and others say he died to the royal magicians' magic. {nl}It is indeed sad news.
+QUEST_20150714_002069	$Propaganda of the Kingdom
+QUEST_20150714_002070	$Surrender by throwing away your weapons.{nl}It is indeed your fault that you've created the Great Earth Fortress to stand against the kingdom, but merciful King Kadumel will save you.
+QUEST_20150714_002071	$Only Premier Eminent, the prime minister of the great kingdom will forgive your treason. {nl}Surrender. You've been misled by the criminal, Ruklys.
+QUEST_20150714_002072	$Hurray to the great kingdom soldiers! {nl}Hurray to the Great Premier Eminent!
+QUEST_20150714_002073	$Even Lydia Schaffen, friend of Ruklys, turned to the side of kingdom soldiers. {nl}No one will stand on your side. {nl}With your hands, collapse The Great Earth Fortress, the proof of your treason, and wave the white flag. {nl}Faithful Premier Eminent shall forgive all.
+QUEST_20150714_002074	$Are you going to die with Ruklys who misled you with malicious tricks to create The Great Earth Fortress to stand against the kingdom? {nl}Or live a new life by serving the king until the end by following the way of Premier Eminent who protected the kingdom from the rebellion?
+QUEST_20150714_002075	$The Records of Rebel Forces
+QUEST_20150714_002076	$As I was retreating from kingdom soldiers, I encountered many demons. {nl}It doesn't make sense. Where did they come from?
+QUEST_20150714_002077	$There's no hope at all. Ah… my lovely Ridell. {nl}The kingdom soldiers did not keep their word. We're all going to die.
+QUEST_20150714_002078	$Those kingdom soldiers! Especially that Premier Eminent! They're no different from demons. {nl}Even the Goddesses would not forgive you.
```

Comment:
> Even the Goddesses would not forgive you.

Maybe this should be "Even the Goddesses would not forgive them."? Them = soldiers and the minister